### ISSUE TITLE
Loading Sound on iOS and Android gives exception AL.LoadAll() needs a current AudioContext

### ISSUE THREAD

**Issue by kennethsuda** [MonoGame/MonoGame#5074]
Using this [sound](https://drive.google.com/file/d/0B9nWV0CYYRuZTW1SdTZKNGZqU2s/view?usp=sharing) I get an error in the Content.Load<SoundEffect>("testsound"). This is using the current Develop branch. It works fine on Windows DX, Windows GL, and Windows UWP.
Android error:
```
Unhandled Exception:

System.InvalidOperationException: AL.LoadAll() needs a current AudioContext.

08-04 16:27:34.451 D/Mono    ( 1378): DllImport attempting to load: '/system/lib/liblog.so'.
08-04 16:27:34.451 D/Mono    ( 1378): DllImport loaded library '/system/lib/liblog.so'.
08-04 16:27:34.451 D/Mono    ( 1378): DllImport searching in: '/system/lib/liblog.so' ('/system/lib/liblog.so').
08-04 16:27:34.451 D/Mono    ( 1378): Searching for '__android_log_print'.
08-04 16:27:34.451 D/Mono    ( 1378): Probing '__android_log_print'.
08-04 16:27:34.451 D/Mono    ( 1378): Found as '__android_log_print'.
08-04 16:27:34.451 I/MonoDroid( 1378): UNHANDLED EXCEPTION:
08-04 16:27:34.451 I/MonoDroid( 1378): System.InvalidOperationException: AL.LoadAll() needs a current AudioContext.
08-04 16:27:34.451 I/MonoDroid( 1378):   at OpenTK.Audio.OpenAL.EffectsExtension..ctor () [0x0001d] in /Users/builder/data/lanes/3540/1cf254db/source/opentk/Source/OpenTK/Audio/OpenAL/AL/EffectsExtension.cs:1220 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_Efx () [0x0000d] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\MonoGameSource\MonoGame\MonoGame.Framework\Audio\OpenALSoundController.cs:305 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Microsoft.Xna.Framework.Audio.OpenALSoundController..ctor () [0x00069] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\MonoGameSource\MonoGame\MonoGame.Framework\Audio\OpenALSoundController.cs:129 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_GetInstance () [0x0000d] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\MonoGameSource\MonoGame\MonoGame.Framework\Audio\OpenALSoundController.cs:297 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Microsoft.Xna.Framework.AndroidGamePlatform..ctor (Microsoft.Xna.Framework.Game game) [0x00082] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\MonoGameSource\MonoGame\MonoGame.Framework\Android\AndroidGamePlatform.cs:28 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Microsoft.Xna.Framework.GamePlatform.PlatformCreate (Microsoft.Xna.Framework.Game game) [0x00001] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\MonoGameSource\MonoGame\MonoGame.Framework\GamePlatform.Mobile.cs:16 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Microsoft.Xna.Framework.Game..ctor () [0x00225] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\MonoGameSource\MonoGame\MonoGame.Framework\Game.cs:68 
08-04 16:27:34.451 I/MonoDroid( 1378):   at TestMonoCommon.Game1..ctor () [0x000c1] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\TestMono\TestMonoCommon\Game1.cs:80 
08-04 16:27:34.451 I/MonoDroid( 1378):   at TestMonoAndroid.Activity1.OnCreate (Android.OS.Bundle bundle) [0x00009] in C:\Users\ksuda\Documents\Visual Studio 2015\Projects\TestMono\TestMonoAndroid\Activity1.cs:22 
08-04 16:27:34.451 I/MonoDroid( 1378):   at Android.App.Activity.n_OnCreate_Landroid_os_Bundle_ (IntPtr jnienv, IntPtr native__this, IntPtr native_savedInstanceState) [0x00011] in /Users/builder/data/lanes/3540/1cf254db/source/monodroid/src/Mono.Android/platforms/android-23/src/generated/Android.App.Activity.cs:2426 
08-04 16:27:34.451 I/MonoDroid( 1378):   at (wrapper dynamic-method) System.Object:d73f327f-bcc4-45f7-8b7c-a6fbc99039f2 (intptr,intptr,intptr)
```

iOS error (using simulator):
```
2016-08-04 16:15:07.963 TestMonoIOS[90534:18603474] 
Unhandled Exception:
System.InvalidOperationException: AL.LoadAll() needs a current AudioContext.
  at OpenTK.Audio.OpenAL.EffectsExtension..ctor () [0x00018] in <filename unknown>:0 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_Efx () [0x0000b] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Audio/OpenALSoundController.cs:305 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController..ctor () [0x00067] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Audio/OpenALSoundController.cs:129 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_GetInstance () [0x0000b] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Audio/OpenALSoundController.cs:297 
  at Microsoft.Xna.Framework.iOSGamePlatform..ctor (Microsoft.Xna.Framework.Game game) [0x0001f] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/iOS/iOSGamePlatform.cs:102 
  at Microsoft.Xna.Framework.GamePlatform.PlatformCreate (Microsoft.Xna.Framework.Game game) [0x00002] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/GamePlatform.Mobile.cs:14 
  at Microsoft.Xna.Framework.Game..ctor () [0x0020d] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Game.cs:68 
  at TestMonoCommon.Game1..ctor () [0x000c1] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoCommon/Game1.cs:80 
  at TestMonoIOS.Program.RunGame () [0x00001] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoIOS/Main.cs:30 
  at TestMonoIOS.Program.FinishedLaunching (UIKit.UIApplication app) [0x00001] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoIOS/Main.cs:62 
  at (wrapper managed-to-native) UIKit.UIApplication:UIApplicationMain (int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, IntPtr principal, IntPtr delegate) [0x00005] in /Users/builder/data/lanes/3234/d8bedd03/source/maccore/src/UIKit/UIApplication.cs:77 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x00038] in /Users/builder/data/lanes/3234/d8bedd03/source/maccore/src/UIKit/UIApplication.cs:61 
  at TestMonoIOS.Program.Main (System.String[] args) [0x00008] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoIOS/Main.cs:53 
2016-08-04 16:15:07.963 TestMonoIOS[90534:18603474] Unhandled managed exception:
AL.LoadAll() needs a current AudioContext. (System.InvalidOperationException)
  at OpenTK.Audio.OpenAL.EffectsExtension..ctor () [0x00018] in <filename unknown>:0 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_Efx () [0x0000b] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Audio/OpenALSoundController.cs:305 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController..ctor () [0x00067] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Audio/OpenALSoundController.cs:129 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_GetInstance () [0x0000b] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Audio/OpenALSoundController.cs:297 
  at Microsoft.Xna.Framework.iOSGamePlatform..ctor (Microsoft.Xna.Framework.Game game) [0x0001f] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/iOS/iOSGamePlatform.cs:102 
  at Microsoft.Xna.Framework.GamePlatform.PlatformCreate (Microsoft.Xna.Framework.Game game) [0x00002] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/GamePlatform.Mobile.cs:14 
  at Microsoft.Xna.Framework.Game..ctor () [0x0020d] in /Users/ksuda/Documents/My Xarmarin Projects/MonoGameSource/MonoGame/MonoGame.Framework/Game.cs:68 
  at TestMonoCommon.Game1..ctor () [0x000c1] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoCommon/Game1.cs:80 
  at TestMonoIOS.Program.RunGame () [0x00001] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoIOS/Main.cs:30 
  at TestMonoIOS.Program.FinishedLaunching (UIKit.UIApplication app) [0x00001] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoIOS/Main.cs:62 
  at (wrapper managed-to-native) UIKit.UIApplication:UIApplicationMain (int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, IntPtr principal, IntPtr delegate) [0x00005] in /Users/builder/data/lanes/3234/d8bedd03/source/maccore/src/UIKit/UIApplication.cs:77 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x00038] in /Users/builder/data/lanes/3234/d8bedd03/source/maccore/src/UIKit/UIApplication.cs:61 
  at TestMonoIOS.Program.Main (System.String[] args) [0x00008] in /Users/ksuda/Documents/My Xarmarin Projects/TestMono/TestMonoIOS/Main.cs:53 
2016-08-04 16:15:07.964 TestMonoIOS[90534:18603474] critical: Stacktrace:
```

**Comment by MacLeek** [comment]
I've got same errors.

**Comment by kennethsuda** [comment]
Fixed by [PR#5086](https://github.com/MonoGame/MonoGame/pull/5086). Tested both Android and iOS.
